.dockerignore

```diff
@@ -0,0 +1,6 @@
+node_modules
```

Comment:
There is no node stuff in the repo at the moment so we probably don't need these or npm-debug.